docs/reference/judoc.md

```diff
@@ -1 +1,93 @@
-# Judoc reference
+# Judoc syntax reference
+Judoc is used to document parts of your code. You can attach *Judoc
+blocks* to the following entities:
+
+1. A module.
+2. A type definition.
+4. A constructor definition.
+3. A type signature of a function.
+4. An axiom definition.
+
+In order to attach documentation to either of these items, write *blocks* of
```

Comment:
You call the items in the list entities in the sentence above, and either usually refers to two alternatives.

```suggestion
In order to attach documentation to any of these entities, write *blocks* of
```

```suggestion
In order to attach documentation to an entity, write *blocks* of
```